.circleci/config.yml

```diff
@@ -529,6 +540,7 @@ workflows:
       - build-linux:
           requires:
             - check
+          context: linux-repo-generation
```

Comment:
What is this and how does it relate to the autoupdater? We should try to avoid introducing new secrets in CircleCI, especially in the build_and_test workflow that runs for all PRs because of security reasons, and because there's a push to get rid of CircleCI eventually anyway -- the fewer secrets we have to migrate over to GitLab CI and GitHub Actions, the easier the migration is going to be.

Reply:
This context contains the private key used to generate a signed `Release` file for the new Ubuntu/Debian package repository I'll be implementing. I was debating whether or not to move this stuff to GitLab or not actually, so we could move it over if you'd like. But currently all of the processing to generate the repo still lives in CircleCI.

Currently it's only a single key so migrating shouldn't be an issue. 0/5 on how we proceed here.

Reply:
From a security perspective I'd be much happier if this never landed on CircleCI. CircleCI has... issues. This is not a hard "no" or anything, but if going directly for GitLab is not too much trouble I'd consider it. I also think the release team will probably have an easier time if we're a bit proactive about moving things over.